pkg/config/config.go

```diff
@@ -253,6 +253,7 @@ type Config struct {
 			PreSignedExpiry               time.Duration `mapstructure:"pre_signed_expiry"`
 			DisablePreSigned              bool          `mapstructure:"disable_pre_signed"`
 			DisablePreSignedUI            bool          `mapstructure:"disable_pre_signed_ui"`
+			DisablePreSignedMultipart     bool          `mapstructure:"disable_pre_signed_multipart"`
```

Comment:
Can we put this under "experimental" or "beta" please?

Reply:
Added configuration reference doc line with **experimental** - didn't want to configure it outside the block adapter.
Let me know if you think we have better place.